packages/amplify-cli/src/index.ts

```diff
@@ -55,12 +54,12 @@ export async function run(): Promise<number> {
     if (e.stack) {
       print.info(e.stack);
     }
-    return 1;
+    process.exit(1);
```

Comment:
Can't we bubble up return code and handle process exit at the top level? is this the best solution?

Reply:
This is actually the top most level, the amplify in the bin directly calls this run method.